implants/lib/c2/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "c2"
+version = "0.0.3"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tonic = { workspace = true }
+prost = { workspace = true}
+prost-types = { workspace = true }
+tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
+
+[build-dependencies]
+tonic-build = "0.10"
+which = "4"
```

Comment:
Haven't seen a single digit version would love to see a more specificity to improve readability and avoid unexpected changes. Eg. `which = "4.0.0` looks like the latest 4 is `4.4.2` not sure which cargo is choosing rn.